### PR TITLE
14 reviewer view review listing frontend

### LINKED ISSUE #14: Reviewer View - 'Review' Listing frontend
## What needs to be done?
- Reviews need to be viewable in the frontend.
- #13 is required before completing this ticket.
- Refers to this page from IndEAAv1:
Screen Shot 2022-06-24 at 1 20 59 pm

## Other
Refers to 23 from our discussion
MicrosoftTeams-image (2)

### PR DESCRIPTION
**Pull request by frinzekt**
## Change Summary
- standardise the format of the user interface
- review listing for the frontend (with typing)
- modularisation of some of the functions
- stubs for the review pages
- addition of new serialiser to get the correct data from the user interface

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
Closes #14

### CODE REVIEW

File: frontend/components/Reviewer/ReviewProgress.tsx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    width: 50,
+    height: 50,
+    display: 'flex',
+    borderRadius: '50%',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  // alternativeLabel: {
+  // 	color: '#FFFFFF',
+  // },
```
Comment (frinzeUWA) [inline frontend/components/Reviewer/ReviewProgress.tsx:62]: unused code

File: frontend/components/Reviewer/ReviewProgress.tsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+  active: {
+    backgroundImage:
+      'linear-gradient( 136deg, rgb(242,113,33) 0%, rgb(233,64,87) 50%, rgb(138,35,135) 100%)',
+    boxShadow: '0 4px 10px 0 rgba(0,0,0,.25)',
+  },
+};
+
+export const StepIcon = (props: { done: Boolean }) => {
+  //   const stepperClasses = useStepperStyles();
```
Comment (frinzeUWA) [inline frontend/components/Reviewer/ReviewProgress.tsx:71]: unused code

File: frontend/components/Reviewer/ReviewProgress.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  return <Box sx={{ ...stepperStyle.root, ...isActiveStyle }}>{icon}</Box>;
+};
+
+type Props = {
+  review: ReviewListEntry;
+  isCoordinator?: Boolean;
+};
+
+const ReviewProgress = ({ review, isCoordinator }: Props) => {
+  //   const stepperClasses = useStepperStyles();
```
Comment (frinzeUWA) [inline frontend/components/Reviewer/ReviewProgress.tsx:84]: unused code

**Review by frinzeUWA — CHANGES_REQUESTED** [review]
remove unused code

File: frontend/components/utils/reviews.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+import { ReviewListEntry } from 'utils/api';
+
+export const determineStepsStateOfReview = (review: ReviewListEntry) => ({
+  step1: Boolean(review.eoc_date_viewed),
+  step2: review.documents.length > 0,
+  step3: review.eoc_specific_reviews.length > 0,
+  step4: Boolean(review.date_submitted),
+});
```
Comment (frinzeUWA) [inline frontend/components/utils/reviews.ts:9]: repeated logic in `ReviewProgress.tsx`

**Review by frinzeUWA — CHANGES_REQUESTED** [review]

**Review by frinzeUWA — APPROVED** [review]
looks good enough

**Comment by frinzekt** [comment]
This is what it looks like now (FYI)
Screen Shot 2022-08-25 at 12 28 56 pm
Screen Shot 2022-08-25 at 12 28 53 pm

**Pull request merged**